.env

```diff
@@ -43,8 +43,14 @@ ASSIGNMENT_STATE_INTERVAL_THRESHOLD=P1D
 LTI_LAUNCH_PRESENTATION_LOCALE=en-EN
 LTI_LAUNCH_PRESENTATION_RETURN_URL=http://localhost:8000/index.html
 LTI_OUTCOME_XML_NAMESPACE=http://www.imsglobal.org/services/ltiv1p1/xsd/imsoms_v1p0
+# Possible values for LTI Version: 1.1.1 or 1.3.0
+LTI_VERSION=1.1.1
 # Possible values for LTI Instance load balancer strategy: username or userGroupId
 LTI_INSTANCE_LOAD_BALANCING_STRATEGY=username
+# Key used for LTI 1.3 link generation - Needed only when LTI_VERSION is set to 1.3.0
+LTI1P3_SERVICE_ENCRYPTION_KEY=lti1p3Key
```

Comment:
you should leave this env surrounded this way:

```
###> oat-sa/bundle-lti1p3 ###
LTI1P3_SERVICE_ENCRYPTION_KEY=xxx
###< oat-sa/bundle-lti1p3 ###
```

this is how SF flex recipes detects that the .env file need update or not :)

Reply:
also, dont hesitate to bring a bit more entropy in this key, it's goal remain to secure, predictable keys should be avoided, whatever environment we target

this applies to all similar cases